mne/time_frequency/tfr.py

```diff
@@ -887,7 +887,8 @@ def __init__(self, info, data, times, freqs, nave, comment=None,
     def plot(self, picks, baseline=None, mode='mean', tmin=None,
              tmax=None, fmin=None, fmax=None, vmin=None, vmax=None,
              cmap='RdBu_r', dB=False, colorbar=True, show=True,
-             title=None, axes=None, layout=None, verbose=None):
+             title=None, axes=None, layout=None, verbose=None,
+             imtype='normal'):
```

Comment:
I know that verbose should be last but `imtype` is temporary - you may for example want me to remove it :)